daemon/api_snaps.go

```diff
@@ -568,7 +568,7 @@ func snapInstallMany(inst *snapInstruction, st *state.State) (*snapInstructionRe
 			return nil, fmt.Errorf(i18n.G("cannot install snap with empty name"))
 		}
 	}
-	installed, tasksets, err := snapstateInstallMany(st, inst.Snaps, inst.userID)
+	installed, tasksets, err := snapstateInstallMany(st, inst.Snaps, inst.userID, &snapstate.Flags{})
```

Comment:
Some of these functions have a check to initialize the flags, like:
```
func InstallMany(..., flags *Flags) {
  if flags == nil {
    flags = &Flags{}
  }
```
This avoids always having to pass flags at every call. We could do the same here

Reply:
I actually added that code, but unit tests complained as flags is no yet being used (see https://github.com/snapcore/snapd/pull/11271/commits/d3ceb93cf47c6be9870b5985c1d4cc099c87bbec). I will add that bit in the continuation PR (there should be no risk of forgetting as the tests I modified pass `nil` to the function).

Reply:
Ah of course, good point :+1: